proposals/lip-0051.md

```diff
@@ -769,14 +770,222 @@ if the above send function does not fail:
+    ],
+    "properties": {
+        "userSubstore": {
+            "type": "array",
+            "fieldNumber": 1,
+            "items": {
+                "type": "object",
+                "required": [
+                    "address",
+                    "tokenID",
+                    "availableBalance",
+                    "lockedBalances"
+                ],
+                "properties": {
+                    "address": {
+                        "dataType": "bytes",
+                        "fieldNumber": 1
+                    },
+                    "tokenID": {
+                        "type": "object",
+                        "fieldNumber": 2,
+                        "required": ["chainID", "localID"],
+                        "properties": {
+                            "chainID": {
+                                "dataType": "uint32",
+                                "fieldNumber": 1
+                            },
+                            "localID": {
+                                "dataType": "uint32",
+                                "fieldNumber": 2
+                            }
+                        }
+                    },
+                    "availableBalance": {
+                        "dataType": "uint64",
+                        "fieldNumber": 3
+                    },
+                    "lockedBalances": {
+                        "type": "array",
+                        "fieldNumber": 4,
+                        "items": {
+                            "type": "object",
+                            "required": ["moduleID", "amount"],
+                            "properties": {
+                                "moduleID": {
+                                    "dataType": "uint32",
+                                    "fieldNumber": 1
+                                },
+                                "amount": {
+                                    "dataType": "uint64",
+                                    "fieldNumber": 2
+                                }
+                            }
+                        }
+                    }
+                }
+            }
+        },
+        "supplySubstore": {
+            "type": "array",
+            "fieldNumber": 1,
+            "items": {
+                "type": "object",
+                "required": ["localID", "totalSupply"],
+                "properties": {
+                    "localID": {
+                        "dataType": "uint32",
+                        "fieldNumber": 1
+                    },
+                    "totalSupply": {
+                        "dataType": "uint64",
+                        "fieldNumber": 1
+                    }
+                }
+            }
+        },
+        "escrowSubstore": {
+            "type": "array",
+            "fieldNumber": 1,
+            "items": {
+                "type": "object",
+                "required": ["escrowChainID", "tokenLocalID", "amount"],
```

Comment:
This should be `localID` to be consistent with definition below.

Reply:
indeed, fixed